oscap_report/html_report/templates/remedations.html

```diff
@@ -32,7 +32,7 @@
                     {% endif %}
                 </tbody>
             </table>
-            <pre>{{ remediation.fix }}</pre>
+            <pre>{{ remediation.fix | replace("<","&lt;") | replace(">","&gt;")}}</pre>
```

Comment:
Is it the only symbol that has to be replaced? What about `&`? TL;DR: https://docs.python.org/3/library/html.html

Reply:
Sorry, that's not Python, it Jinja. Nonetheless: https://jinja.palletsprojects.com/en/2.10.x/templates/#html-escaping